### PR TITLE
Updated Safety Resources sub module by code comments to its method and class

### PR DESCRIPTION
**Pull request by minervapanda**
Clean up code by structuring code comments for class and methods of Safety Resources

### CODE REVIEW

**Comment by sandarumk** [comment]
There are two classes in the Safety Resources sub module. Add method comments too according to the template discussed there. You can use the files of circle of trust as a reference.

**Comment by minervapanda** [comment]
Just did the both. let me know in case of any changes @sandarumk

**Comment by minervapanda** [comment]
I am unable to find the author name. Is it the one who does the last commit ?

**Comment by sandarumk** [comment]
Limit this to one commit per sub-module. There is a large number of commits here. Most of them with the same commit message. Squash them in to one. You can find the author by going through the history of the file. The author is not the last one who updated it. It is the first one who created it.

File: app/src/main/java/com/peacecorps/pcsa/safety_resources/SafetyResources.java [hunk 1]
```diff
@@ -7,9 +7,16 @@
...
-
+/*
+ * Activity for Safety Resources fragment container
+ *
+ * @author calistus
+ * @since 2015-08-18
+ */
 public class SafetyResources extends AppCompatActivity {
-
+/**
```
Comment (sandarumk) [inline app/src/main/java/com/peacecorps/pcsa/safety_resources/SafetyResources.java]: Comments should be aligned.

**Comment by minervapanda** [comment]
Please check if it is okay to be merged

**Comment by sandarumk** [comment]
Your PR now has 7 commits.

**Comment by rohands** [comment]
@minervapanda This could help :)
https://github.com/blog/2141-squash-your-commits

**Comment by yogeshbalan** [comment]
@rohands I think that link is describing the squash feature for repository administrator and not for the contributor. But it is clearly helpful to understand squashing concept.
@minervapanda , this might help as well. [link](http://blog.steveklabnik.com/posts/2012-11-08-how-to-squash-commits-in-a-github-pull-request)

**Comment by minervapanda** [comment]
I changed the "pick" to squash . and pushed and the repo is up to date now . But the pull requests are not in squashed form.

**Comment by yogeshbalan** [comment]
@minervapanda .. you need to close this PR and send a new one.

**Pull request closed without merging**